pkgs/development/python-modules/django-filter/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, django
+, djangorestframework, python, mock
+}:
+
+buildPythonPackage rec {
+  pname = "django-filter";
+  version = "2.1.0";
```

Comment:
Out of date
https://github.com/carltongibson/django-filter/releases/tag/2.2.0

Reply:
fixed